### PR TITLE
AppInfoView: replace combobox with dropdown

### PR DESCRIPTION
**Pull request by danirabbit**
Knocks out a couple deprecation warnings by using dropdown and Glib.ListStore

### CODE REVIEW

**Review by leolost2605 — CHANGES_REQUESTED** [review]
Works as it should however when clicking the dropdown button its background becomes transparent so that it has the banner background color

**Comment by danirabbit** [comment]
@leolost2605 fixed! Just a reminder, you need to do `ninja clean` to rebuild CSS

File: src/Views/AppInfoView.vala [hunk 1]
```diff
@@ -1218,6 +1202,25 @@ public class AppCenter.Views.AppInfoView : AppCenter.AbstractAppContainer {
...
+        list_item.set_data ("title", title);
+        list_item.set_child (title);
+    }
+
+    private void origin_bind_factory (Object object) {
+        var list_item = object as Gtk.ListItem;
+
+        var package = (AppCenterCore.Package) list_item.get_item ();
+
+        var title = list_item.get_data<Gtk.Label> ("title");
```
Comment (leolost2605) [inline src/Views/AppInfoView.vala:1220]: I think instead of doing this manually you can just cast the `list_item.child` as a label

**Review by leolost2605 — CHANGES_REQUESTED** [review]

File: src/Views/AppInfoView.vala (hunk 1 above)
Comment (danirabbit) [reply src/Views/AppInfoView.vala]: You got it!

**Comment by danirabbit** [comment]
@leolost2605 can I get a re-review? :)

**Review by leolost2605 — APPROVED** [review]
Sorry kinda forgot about this 😔

LGTM!

**Pull request merged**